doc/overview.rst

```diff
@@ -41,40 +44,39 @@ And a list 3rd party libraries (which you can find the installation chapter).
     When you use Composer to install and update dependencies the
     installation of these libraries will be handled for your.
 
-Depending on your needs, you may need the following:
-
-* `International <http://www.php.net/manual/en/book.intl.php>`_ support when using Date/Time
-* `BCMath <http://php.net/manual/en/book.bc.php>`_ or `GNU Multiple Precision (GMP) <http://php.net/manual/en/book.gmp.php>`_ for handling big numbers
-* `Doctrine ORM <http://www.doctrine-project.org/projects/orm.html>`_ for database support
-
 Component Breakdown
 -------------------
 
-The Search Component is made up of many classes. Each of these classes can be grouped
-into a general "component" group which describes the task it is designed to
+RollerworksSearch is made up of many classes. Each of these classes can be grouped
+into a general "component group" which describes the task it is designed to
 perform.
 
-We'll take a brief look at the components which form the Search Component as a whole,
+We'll take a brief look at the components which form RollerworksSearch as a whole,
 in this section of the book.
 
-ValuesGroup and ValuesBag
-~~~~~~~~~~~~~~~~~~~~~~~~~
+SearchCondition
+~~~~~~~~~~~~~~~
+
+Each search operation starts with a SearchCondition (``SearchConditionInterface``)
+consisting of a ``ValuesGroup`` and ``FieldSet`` object.
 
-At the root of each search there is at least one ``ValuesGroup`` object, containing
-the field names with there values (as a ``ValuesBag`` object), and optionally subgroups
+At the root of each SearchCondition is a ``ValuesGroup`` object, containing
+the values (as ``ValuesBag`` object) per field name and *optionally** subgroups
 (each one being a ``ValuesGroup`` object).
 
-A ``ValuesGroup`` is 'logically' marked as AND by default, meaning that the search
-condition will only be true if from each field inside the group a value is true (matching).
-But it is however possible to mark a group as OR, meaning that at least one field must match and
-other fields are considered optional.
+A ``ValuesGroup`` is 'logically' marked as AND by default meaning that the search
```

Comment:
Compare this to an SQL where clause `WHERE id=1 AND date='2015-02-04'`.
The id and date are both separate fields but you only get a result if both of the fields evaluate to true (are matching).